### ISSUE TITLE
Error loading warps(1.20.1)

### ISSUE THREAD

**Issue by Andrey2345Z** [LeraRiemann/ProjectInfinity#37]
At one point I decided to travel through the warps of this mod, but when I tried to use the /warp command, the command sent me to warps from 0 to 10, which were always the same (as if scripted). A command with other warps, starting from 11, produced an error. When I took a random book from an endless bookshelf and threw it into the portal, the game crashed completely with the error code "-1" and the text:

The game crashed while colliding entity with block
Error: java.lang.NullPointerException: Cannot invoke "java.lang.Double.doubleValue()" because the return value of "java.util.Map.get(Object)" is null

I will also attach a bug report file
[crash-2024-06-25_21.57.32-server.txt](https://github.com/user-attachments/files/15977740/crash-2024-06-25_21.57.32-server.txt)

I don’t understand what the problem is, given that I didn’t install any mods (although I once tested the mod with Optifabric (Optifine on fabric) and XaerosWorldMap)

**Comment by LeraRiemann** [comment]
seems like a generic case of not deleting config files prior to updating the mod's version, should be handled automagically in latest mod versions?

you can reopen if the behaviour persists

**Comment by Andrey2345Z** [comment]
Errors in loading warps still appear, but this time they appear immediately. I managed to launch warps several times, until at one point the game stopped executing commands and gave an error with the text:

"Unbound values ​​in registry"
"ResourceKey[minecraft:root / minecraft:worldgen/biome]:[infinity:biome_216964690, infinity:biome_606208577]"

When I tried to get to the warp I needed (after the command error) through the portal, the game crashed, with the same text above

The funny thing is that the game complains about different biomes, and not only about biome 216964690 and biome 606208577, the game complains about those biomes that I have already visited in this game session. Which makes me feel like the game works until it loads what it considers to be "broken" biomes, which causes an error in loading other warps. What's even funnier is that after rebooting the launcher, the game was able to load the warp I needed, but at one point it started to crash again and wouldn't let me into even the warps I'd already visited before, giving the same error, but with other mod biomes. And this keeps repeating itself.
